package.json

```diff
@@ -72,6 +72,8 @@
     "eslint-plugin-disable-features": "^0.1.3",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.3",
+    "ember-engines": "^0.8.5",
+    "eager-engine": "link:./tests/dummy/lib/eager-engine",
```

Comment:
I don't think we need to add `ember-engines` as a dependency here.

Reply:
why not @rwjblue? how can I create a engine-repo without ember-engines addon?

Reply:
I'm following the @ef4 tip to create an engine, instead make engine dynamically (see the comment https://github.com/emberjs/ember-test-helpers/pull/795#issuecomment-615465238)